Dockerfile

```diff
@@ -1,4 +1,16 @@
-FROM python:3.4-onbuild
+# https://github.com/docker-library/official-images/issues/2076
+FROM python:3.4
+
+RUN pip install pipenv --upgrade
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+COPY Pipfile Pipfile
```

Comment:
Do we need to copy the `Pipfile` in for the pipenv install, or will it work with just the `Pipfile.lock`?

Reply:
I was using the [Pipenv Dockerfile](https://hub.docker.com/r/kennethreitz/pipenv/~/dockerfile/) as an example but I think that was only so the developer is able to play around with the full functionality. It's probably unnecessary for us.

Reply:
Turns out you do need at the very least a Pipfile otherwise Pipenv fails hard on Docker when it's unable to create one.